CHANGES.md

```diff
@@ -6,6 +6,25 @@ list](https://github.com/trinodb/trino-python-client/tags), the
 [README](https://github.com/trinodb/trino-python-client/blob/master/README.md)
 and the [PyPI page](https://pypi.org/project/trino/).
 
+## Trino Python client 0.317.0
```

Comment:
How about just `Release 0.317.0`? Repeating `Trino Python client` in Trino Python client looks redundant. 

No need to change in this PR since `Trino Python client 0.316.0` already exists.